CONTRIBUTING.md

```diff
@@ -29,6 +29,17 @@ To contribute to The Boring JavScript Stack, follow these steps:
 
 ---
 
+## Creating an Issue
+Before **creating** an issue, please follow these steps:
+
+1. Search existing issues before creating a new issue (has someone created an issue already).
```

Comment:
Cool, can we rephrase it to do this @d-a-ve  👇🏾
Before creating a new issue, it is advisable to search for any existing issues that may have already been created.